source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/ChargeLinks/Handlers/ChargeLinksCommandHandlerTests.cs

```diff
@@ -29,14 +32,18 @@ public class ChargeLinksCommandHandlerTests
         [Theory]
         [InlineAutoDomainData]
         public async Task HandleAsync_WhenCalledWithValidChargeLink_ShouldReturnOk(
```

Comment:
And this test name doesn't seem to match the test. How about:
```suggestion
        public async Task HandleAsync_WhenValidChargeLinkCommand_DispatchesOnce(
```